docs/reference/indices/analyze.asciidoc

```diff
@@ -96,6 +96,24 @@ coming[2.0.0, body based parameters were added in 2.0.0]
 Will cause the analysis to happen based on the analyzer configured in the
 mapping for `obj1.field1` (and if not, the default index analyzer).
 
+If `detail` parameter is `true`, this API output each tokens within tokenizer chain and output with all token attributes.
+It also output only token attribute specified by `attributes`.
+If `short_attr_name` is `false`, it output token attribute full name.
```

Comment:
I would write this like:

If you want to get more advanced details, set `detail` to `true`  (defaults to `false`). It will output all token attributes for each token.
You can filter token attributes you want to output by setting `attributes` option.
By default, token attributes are printed using their short names but you can display the long name by setting `short_attr_name` to `false` (defaults to `true`).